ca/config.go

```diff
@@ -327,6 +336,14 @@ func RenewTLSConfig(ctx context.Context, s *SecurityConfig, baseCertDir string,
 				updates <- CertificateUpdate{Err: err}
 			}
 
+			// Update the external CA to use the new client TLS
+			// config using a copy without a serverName specified.
+			s.externalCA.UpdateTLSConfig(&tls.Config{
+				Certificates: clientTLSConfig.Certificates,
+				RootCAs:      clientTLSConfig.RootCAs,
+				MinVersion:   tls.VersionTLS12,
```

Comment:
We should probably take both MinVersion and ciphers from `clientTLSConfig` so in the future we only have 1 config we have to change. Again, ideally we wouldn't have an independent CA TLSConfig